### PR TITLE
refactor(docs): update building.rst to include Visual Studio 2022 building instructions

### PR DESCRIPTION
**Pull request by electro-logic**

### CODE REVIEW

File: doc/building.rst [hunk 1]
```diff
@@ -69,17 +77,40 @@ CMake project definition looks as follows:
     #   find_package(open62541 REQUIRED)
     #   target_link_libraries(main open62541::open62541)
 
-Building with CMake on Windows
+Building with Visual Studio on Windows
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (neojaw) [inline doc/building.rst:81]: The underline must/should be as long as the heading, according to rst. Sphinx will still build it but print the warning `Title underline too short`.

File: doc/building.rst [hunk 2]
```diff
@@ -69,17 +77,40 @@ CMake project definition looks as follows:
...
+
+.. code-block:: bash
 
-  - Python 3.x: https://python.org/downloads
-  - CMake: http://www.cmake.org/cmake/resources/software.html
-  - Microsoft Visual Studio: https://www.visualstudio.com/products/visual-studio-community-vs
+    msbuild open62541.sln /v:n -t:rebuild -m
+
+Building with CMake/MinGW on Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (neojaw) [inline doc/building.rst:106]: Ditto.

Comment (electro-logic) [reply doc/building.rst]: good catch, I'm new to Sphinx and I misssed that

**Comment by jpfr** [comment]
👍

**Pull request merged**